static/src/javascripts/projects/commercial/modules/messenger/viewport.js

```diff
@@ -71,8 +70,4 @@ const onMessage = (respond: any, start: any, iframe: ?Element): void => {
     }
 };
 
-register('viewport', onMessage, {
-    persist: true,
-});
-
-export { addResizeListener, removeResizeListener, reset };
+export { addResizeListener, removeResizeListener, reset, onMessage };
```

Comment:
Since `onMessage` is the only function that is imported into our application code, can we expose the rest of the functions using an `_` export, to convey the intention that they are only used in tests?